dev/src/reference.ts

```diff
@@ -60,7 +58,6 @@ const comparisonOperators:
     {[k: string]: api.StructuredQuery.FieldFilter.Operator} = {
       '<': 'LESS_THAN',
       '<=': 'LESS_THAN_OR_EQUAL',
-      '=': 'EQUAL',
       '==': 'EQUAL',
```

Comment:
Per discussions with Gil, etc. during the push to GA, I think we want to continue to allow '=' to avoid the breaking change.